### ISSUE TITLE
[META] endless loop when trying to create a feature request issue on github

### ISSUE THREAD

**Issue by mr-feek** [japa/runner#36]
### Package version

N/A

### Describe the bug

This isn't a bug with `japa` itself, but rather the repository issue configuration. 

**Steps to reproduce:**:
- visit https://github.com/japa/runner/issues 
- click `New Issue`, be brought to https://github.com/japa/runner/issues/new/choose 
- click `Feature Request`

**Actual Functionality**:
- You are brought to the exact same issue selection page in a new tab.

**Expected Functionality**
- You are able to file a feature request issue

So basically its currently not possible to file a feature request issue with the current UI, you just end up back on the issue type selection page.

**Comment by thetutlage** [comment]
@McSneaky Do you face the same issue. Because, when I try clicking on any of the links it just works fine

**Comment by McSneaky** [comment]
@thetutlage jup

https://github.com/japa/runner/assets/6796697/9de8fbf1-5da6-4c2b-b34d-9a2e2c2fdf30
